### ISSUE TITLE
Menubar icon looks weird

### ISSUE THREAD

**Issue by danini-the-panini** [pablopunk/SwiftShift#63]
Compared to other icons in my menubar, SwiftShift seems to not fit in, I'm not sure if it's supposed to look like that

<img width="177" alt="image" src="https://github.com/pablopunk/SwiftShift/assets/1171825/b358ba4f-1cba-43f7-8155-2d1c5930ef68">

**Comment by pablopunk** [comment]
yep, that's how it looks, but i see what you mean, i could make it bolder or even invert the color & empty space

**Comment by pablopunk** [comment]
WDYT? left: new, right: old

<p><img width="400" src="https://github.com/pablopunk/SwiftShift/assets/4324982/5940064c-7eb9-4fef-8b6c-440127535715" /></p>
<p><img width="400" src="https://github.com/pablopunk/SwiftShift/assets/4324982/31b05335-7572-499e-85b8-1073ad5bcc5f" /></p>

**Comment by danini-the-panini** [comment]
much better :)

**Comment by pablopunk** [comment]
@danini-the-panini I realized that I never released this. It should be included in the latest update. 🙌🏻
